src/main/java/fourstep/logic/Car.java

```diff
@@ -0,0 +1,33 @@
+package fourstep.logic;
+
+
+public class Car {
+
+    private static final int THRESHOLD = 4;
+    private int location = 1;
+    private final String name;
```

Comment:
```suggestion
    private static final int THRESHOLD = 4;

    private int location = 1;
    private final String name;
```

상수와 인스턴스 변수 사이에는 한 줄 띄워 주세요. 또한, 상수명은 조금 더 구체적이어도 좋을 것 같아요.